filters/auth/grant.go

```diff
@@ -0,0 +1,239 @@
+package auth
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+	"golang.org/x/oauth2"
+)
+
+const (
+	OAuthGrantName = "oauthGrant"
+
+	bearerPrefix              = "Bearer "
```

Comment:
this is a duplicate of `authHeaderPrefix`

Reply:
Removed. It was unused.